components/dashboard/sections/accounting/AccountingCategoryDrawer.tsx

```diff
@@ -60,31 +72,48 @@ type AccountingCategoryDrawerViewProps = {
   accountingCategory?: Pick<AccountingCategory, 'id' | EditableAccountingCategoryFields>;
   onEditClick: () => void;
   onDeleteClick: () => void;
+  hostSlug: string;
 };
 
 function AccountingCategoryDrawerView(props: AccountingCategoryDrawerViewProps) {
   const intl = useIntl();
+
+  const query = useQuery<AccountingCategoryDrawerViewQuery, AccountingCategoryDrawerViewQueryVariables>(
+    gql`
+      query AccountingCategoryDrawerView($hostSlug: String!) {
+        host(slug: $hostSlug) {
+          id
+          type
+        }
+      }
+    `,
+    {
+      context: API_V2_CONTEXT,
+      variables: {
+        hostSlug: props.hostSlug,
+      },
+    },
+  );
+
+  if (query.loading) {
+    return <Loading />;
+  }
+
+  const isIndependentCollective = query.data.host.type === CollectiveType.COLLECTIVE;
+
```

Comment:
I understand that this will use the Apollo cache, but for the simplicity of the code couldn't we just pass down the `isIndependentCollective` prop from `AccountingCategoriesTable`?